.github/CONTRIBUTING.md

```diff
@@ -54,8 +54,25 @@ Write [meaningful commit messages](http://who-t.blogspot.de/2009/12/on-commit-me
 
 Good commit messages speed up the review process and help to keep this project maintainable in the long term.
 
+## Jenkins credential handling
```

Comment:
This is more the less the same what already have here:

> Variables, methods, types and so on shall have meaningful self describing names.

Any reason for repeating this here for the Jenkins credentials? Not against this, just curious.

Reply:
just saw the discussion in #213 
This is to close this as per the issue ...

Reply:
and it is a more concrete guiding principle based on what we have ...